lib/skate_web/controllers/route_tabs_controller.ex

```diff
@@ -4,6 +4,12 @@ defmodule SkateWeb.RouteTabsController do
   alias SkateWeb.AuthManager
   alias Skate.Settings.RouteTab
 
+  def create(conn, %{"route_tab" => route_tab} = _params) do
+    username = AuthManager.Plug.current_resource(conn)
+    RouteTab.create_for_user!(username, format_tab(route_tab))
+    json(conn, %{})
+  end
```

Comment:
**question:** how is this actually invoked from the front end? It isn't mapped in `router.ex` and I also don't see any changes to `api.ts`.

Reply:
It's not...that can come out.